### PR TITLE
Add LS changes to 2201.2.0 release note

### PR DESCRIPTION
**Pull request by mindula**
## Purpose
Add LS release note for the 2201.2.0 release.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md [hunk 1]
```diff
@@ -196,13 +196,24 @@ digraph "org/package:0.1.0" {
 
 #### New features
 
+##### Language Server
+
+- Added API docs reference support on hover
+- Added new code actions to extract anonymous records into records and to generate undefined record types
+- Introduced new code actions to generate getters and setters for class-level variables
+- Added a new code action to make annotation declarations with the 'source' attach point(s) constant
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md:204]:
```suggestion
- Added a new code action to make annotation declarations with the `source` attach point(s) constant
```

File: release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md [hunk 2]
```diff
@@ -196,13 +196,24 @@ digraph "org/package:0.1.0" {
...
+
 ##### OpenAPI Tool
 Added support for generating client resource methods in the client generation command. The preferred client method type can be chosen using the `--client-methods=<remote(default)|resource>` option.
   - `$ bal openapi -i <OpenAPI contract> --client-methods=resource`
   - `$ bal openapi -i <OpenAPI contract> --mode client --client-methods=resource`
 
 #### Improvements
 
+##### Language Server
+- Improved the `Create variable` code action in async send action
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md:214]:
```suggestion
- Improved the `Create variable` code action in the `async send` action
```

File: release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md [hunk 3]
```diff
@@ -196,13 +196,24 @@ digraph "org/package:0.1.0" {
...
+- Added completion support for resource access action context
```
Comment (praneesha) [inline release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md:215]:
```suggestion
- Added completion support for the `resource access` action context
```

File: release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md [hunk 4]
```diff
@@ -196,13 +196,24 @@ digraph "org/package:0.1.0" {
 
 #### New features
 
+##### Language Server
+
+- Added API docs reference support on hover
+- Added new code actions to extract anonymous records into records and to generate undefined record types
+- Introduced new code actions to generate getters and setters for class-level variables
```
Comment (gayalkuruppu) [inline release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md:203]: Shall we mention that this is a `source action`?  And should also mention source actions such as this and `Optimize all imports` are not shown directly under the bulb but under `source actions` option in the right click menu?

File: release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md [hunk 5]
```diff
@@ -196,13 +196,24 @@ digraph "org/package:0.1.0" {
 
 #### New features
 
+##### Language Server
+
+- Added API docs reference support on hover
```
Comment (IMS94) [inline release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md:201]: Let's elaborate this a bit more. 
```suggestion
- Added API docs reference support on hover. Now when you hover over a construct (class, type, function) the hover documentation will include a link to view the API docs specific to that construct.
```

**Review by gayalkuruppu — CHANGES_REQUESTED** [review]

**Review by IMS94 — CHANGES_REQUESTED** [review]

File: release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md (hunk 4 above)
Comment (mindula) [reply release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md]: Done

**Review by praneesha — APPROVED** [review]

**Review by IMS94 — APPROVED** [review]

**Review by mohanvive — APPROVED** [review]

**Review by gayalkuruppu — APPROVED** [review]

File: release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md (hunk 3 above)
Comment (sanjana) [reply release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md]: @mindula, please refer to this.
https://github.com/ballerina-platform/ballerina-release/pull/2094#discussion_r946328832

Comment (mindula) [reply release-notes/swan-lake-release-notes/swan-lake-2201.2.0-release-note.md]: Fixed in #2095

**Pull request merged**